beavy/common/paging_schema.py

```diff
@@ -17,11 +17,11 @@ def move_to_meta(self, data):
         # FIXME: add support for paging links
         return {
             "meta": data,
-            "data": items.get("data", []),
-            "links": items.get("links", [])
```

Comment:
It's supposed to hold paging links, but since we are using infinite pagination it's not implemented yet. This will also be a bit challenging because it's going to be hard for the pagination object to know, from the front-end, the details of the type of pagination.